CONTRIBUTING.md

```diff
@@ -141,6 +144,57 @@ to email the patch to <gap@gap-system.org>.  You can refer to
 [this tutorial](https://burzalodowa.wordpress.com/2013/10/05/how-to-send-patches-with-git-send-email/)
 on how to do this.
 
+## The reviewing process
+
+Before a change can be incorporated into the code base, our setup requires
+each pull request (PR) to be
+[reviewed](https://help.github.com/articles/about-pull-request-reviews/)
+by at least one GAP developer (i.e. anybody with write access to the
+[gap repository](https://github.com/gap-system/gap/)).
```

Comment:
Hmm "our setup requires" sounds as if it wasn't a conscious decision, but rather is an accident of technical requirement

And a "GAP developer" is anybody with write access... ? I am not sure I'd agree :-). Also, there does not seem a way to find out who has write access to a repository, so this is not all that helpful in practice either. How about something more like this:

> Before any change is incorporated into the GAP code base, it must undergo a mandatory code review.
> Typically, this is done for each pull request (PR) via the
> [GitHub code review facilitates]((https://help.github.com/articles/about-pull-request-reviews/)).
> The PR must have at least one approving code review from a core GAP developer with write access to
> the GAP code repository. However, everybody is very welcome to submit code reviews. This helps
> the core developers a lot, and is a step towards become one of them yourself.
>
> To do so, search ...

Reply:
Ah, then that was my "obvious" definition which was only in my head and apparently also wrong. :)